src/PeclEvLoop.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+namespace React\EventLoop;
+
+use Ev;
+use EvLoop;
+use EvIo;
+use EvTimer;
+use EvPeriodic;
+use React\EventLoop\Tick\FutureTickQueue;
+use React\EventLoop\Tick\NextTickQueue;
+use React\EventLoop\Timer\Timer;
+use React\EventLoop\Timer\TimerInterface;
+use SplObjectStorage;
+
+/**
+ * @see https://bitbucket.org/osmanov/pecl-ev/overview
+ */
+class PeclEvLoop implements LoopInterface
+{
+    private $loop;
+    private $nextTickQueue;
+    private $futureTickQueue;
+    private $timerEvents;
+    private $readEvents = [];
+    private $writeEvents = [];
+    private $running;
+
+    public function __construct()
+    {
+        $this->loop             = EvLoop::defaultLoop();
```

Comment:
because it is imported in line 6. https://github.com/reactphp/event-loop/pull/25/files#diff-153b21d8366051a7e96b20134e155fa8R6